admin/sync_logs.py

```
@@ -0,0 +1,218 @@
+            except json.JSONDecodeError as e:
+                print(f"JSON decode error on line {line_num}: {e}")
+                continue
+            except Exception as e:
+                print(f"Error processing line {line_num}: {e}")
+                continue
+
+    # Commit changes
+    conn.commit()
+    conn.close()
+
+    print(f"Sync completed:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)